application/common/components/adapters/WorkdayIdStore.php

```diff
@@ -121,6 +121,18 @@ public function getAllActiveUsers()
                 var_export($allActiveUsers, true)
             ), 1532982679);
         }
+
+        foreach ($allActiveUsers as $key => $user) {
+            $companyIDs = str_replace(" ", ",", $user["company_ids"] ?? "");
+            $ouTree = str_replace(" ", ",", $user["ou_tree"] ?? "");
+            if (strlen($companyIDs) > 0 && strlen($ouTree) > 0) {
```

Comment:
Nice approach, here! That's a clever way to only add the comma when it's needed. :-)